README.KR.md

```
@@ -15,14 +15,14 @@
 <p align="center" style="font-size: 12px;">
-    <a href="https://docs.langinfra.org" style="text-decoration: underline;">문서</a> -
+    <a href="https://docs-langinfra.khulnasoft.com" style="text-decoration: underline;">문서</a> -
     <a href="https://discord.com/invite/EqksyE2EX9" style="text-decoration: underline;">Discord에 참여하기</a> -
     <a href="https://twitter.com/langinfra_ai" style="text-decoration: underline;">X에서 팔로우하기</a> -
     <a href="https://huggingface.co/spaces/Langinfra/Langinfra" style="text-decoration: underline;">실시간 데모</a>
 </p>
 
 <p align="center">
-    <a href="https://github.com/khulnasoft/langinfra">
+    <a href="https://github.com/langinfra/langinfra">
         <img src="https://img.shields.io/github/stars/khulnasoft/langinfra">
```

Markdownlint (0.37.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)